tests/checkpointing/test_model_checkpoint.py

```diff
@@ -1253,7 +1234,7 @@ def test_save_last_saves_correct_last_model_path(tmpdir):
     trainer = Trainer(callbacks=mc)
     trainer.strategy.connect(BoringModel())
 
-    mc._save_last_checkpoint(trainer, {"foo": 1})
+    mc._save_last_checkpoint(trainer, {"foo": 1, "step": 0})
```

Comment:
This isn't a breaking change for the users. Only necessary here because it's an unit test